plugin/nuuid.vim

```diff
@@ -7,23 +7,36 @@ if !exists('g:nuuid_case')
 	let g:nuuid_case = "lower"
 endif
 
-" Make sure we have python
-if !has('python')
-	" finish
-endif
```

Comment:
Deleted this block because it doesn't do anything. (the `finish` is commented out, and anyway we don't *need* python)